MatrixSDKExtensions/VoIP/Jingle/MXJingleCallStackCall.m

```diff
@@ -434,6 +416,8 @@ - (void)peerConnection:(RTCPeerConnection *)peerConnection
 {
     NSLog(@"[MXJingleCallStackCall] didRemoveStream");
     
+    //  TODO: Check this stream has the main stream ID (kMXJingleCallWebRTCMainStreamID) after all platforms decided to use the same stream id.
```

Comment:
Not sure that the IDs should really matter, only how many are left. We don't really care about stream ID until we start dealing with multiple video or audio streams.

Reply:
Updated